basics/src/test/java/basics/CarTests.java

```diff
@@ -0,0 +1,15 @@
+import org.junit.Assert;
+import org.junit.Test;
+
+//@Test
+public class CarTests {
+	
+	@Test
+	public void canBuildCar() {
+		Car car = new Car("Corvette");
+		
+		Assert.assertNotNull(car);

```

Comment:
Put test sections into order by Arrange, Act, Assert

Reply:
Suggested changes were made. Should I close and make another pull request or continue with this?

Reply:
Keep this one